src/tz.cpp

```diff
@@ -2128,7 +2116,11 @@ time_zone::init() const
 sys_info
 time_zone::load_sys_info(std::vector<detail::transition>::const_iterator i, sys_seconds tp) const
 {
-    auto load_posix_info = [&](){auto tz = Posix::time_zone{ posix_timezone_ }.get_info(tp); if (tz.begin < transitions_.back().timepoint){tz.begin = transitions_.back().timepoint;} return tz;};
+    auto load_posix_info = [&](){
```

Comment:
This just looks like a formatting change. Should this be dropped to avoid changing more than we need for future upgrades?